### DISCUSSION TITLE
voice format

### DISCUSSION THREAD

**Discussion by fatemehkhoramdel** [deepgram/community#1010] [Feature Requests]
I am using deepgram to transcript voice and with some device i got error like sumsung a13 or Xiaomi. even i tried to convert voice and then send to deepgram but it doesn't work

**Comment by deepgram-community** [comment]
Hey there! It looks like you haven't connected your GitHub account to your Deepgram account. You can do this at https://community.deepgram.com - being verified through this process will allow our team to help you in a much more streamlined fashion.

**Comment by deepgram-community** [comment]
It looks like we're missing some important information to help debug your issue. Would you mind providing us with the following details in a reply?

- The programming language you are working in (e.g. JavaScript, Python).
- A request ID that triggered your error or issue.

**Reply by fatemehkhoramdel** [reply]
The programming language is Python and about request ID I don't know
